R/waterfall.R

```diff
@@ -273,7 +273,8 @@ waterfall <- function(x, mainRecurCutoff=0, mainGrid=TRUE, mainXlabel=FALSE,
     data_frame <- waterfall_NA2gene(data_frame)
 
     # Plot the Heatmap
-    plot_x_title <- !is.null(clinData) && !plot_proportions
+    #plot_x_title <- !is.null(clinData) && !plot_proportions
+    plot_x_title <- ifelse(!is.null(clinData) || plot_proportions, FALSE, TRUE)
```

Comment:
This line would be simpler as `plot_x_title <- !(!is.null(clinData) || plot_proportions)`